test/acceptance/tests/connect/connect_inject_test.go

```diff
@@ -111,6 +112,7 @@ func TestConnectInject_CleanupController(t *testing.T) {
 			ctx := suite.Environment().DefaultContext(t)
 
 			helmValues := map[string]string{
+				"global.imageK8S":              "kschoche/consul-k8s-dev2",
```

Comment:
This will be removed as soon as the cleanup controller PR is merged.

Reply:
🙏 could we instead set it in the `.circleci/config.yaml` for the `acceptance` job so we're using the latest image for all tests?

Reply:
Definitely can do, great idea!